old-scala-codegen/BUILD

```diff
@@ -0,0 +1,30 @@
+load("@io_bazel_rules_scala//scala:scala.bzl", "scala_library")
+
+scala_library(
```

Comment:
So you split `codegen` in two? What's the difference between `codegen` and `old-scala-codegen\?

Reply:
old-scala-codegen is the codegen as it was previously in this repo. codegen is copied from bsp-generators